content/docs/command-reference/repro.md

```diff
@@ -36,9 +37,10 @@ and caches relevant <abbr>data artifacts</abbr> along the way.
 💡 For convenience, a Git hook is available to remind you to `dvc repro` when
 needed after a `git commit`. See `dvc install` for more details.
 
-There's a few ways to restrict the stages that will be regenerated by this
-command: by specifying stage file `targets`, or by using the `--single-item`,
-`--cwd`, or other options.
+There are a few ways to restrict the stages that will be regenerated by this
+command: by specifying stage `targets`( can be a `.dvc` file or one of the
+stages in `dvc.yaml` file ), or by using the `--single-item`, `--cwd`, among
```

Comment:
.dvc files are not stages, please rephrase. Feel free to break up the sentences, it's all about the writing. BTW, strange formatting again (spaces around paragraphs, and missing after comma) — please keep in mind this is a writing program. We'd prefer not having to worry much about copy editing GSoD-related contributions. Thanks

Reply:
> BTW, strange formatting again (spaces around paragraphs, and missing after comma) — please keep in mind this is a writing program. We'd prefer not having to worry much about copy editing GSoD-related contributions.

I would keep that in mind from the next commits.

> .dvc files are not stages, please rephrase.

Maybe we should change the sentence to - 

"There are a few ways to restrict the stages that will be regenerated by this command: by specifying `targets`, or by using the --single-item, --cwd, among other options."

As targets are already defined in the "Synopsis", specifying `targets` here would by default mean that .dvc files are also accepted. WDYT @jorgeorpinel ?

Reply:
> Maybe we should change the sentence to "... by specifying targets, or by using the --single-item, --cwd, among other options." As targets are already defined in the "Synopsis"

It's a good idea but the synopsis isn't enough. We want to explain everything completely in the description again. It's OK if it repeats the text from the Synopsis, for example.

Reply:
> Maybe we should change the sentence to "... by specifying targets, or by using the --single-item, --cwd, among other options." As targets are already defined in the "Synopsis"

It's a good idea but the synopsis isn't enough. We want to explain everything completely in the description again. It's OK if it repeats the text from the Synopsis, for example.